src/stream/src/executor/hash_agg.rs

```diff
@@ -184,6 +190,8 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
                 group_change_set: HashSet::new(),
                 lookup_miss_count: AtomicU64::new(0),
                 total_lookup_count: AtomicU64::new(0),
+                chunk_lookup_miss_count: AtomicU64::new(0),
+                chunk_total_lookup_count: AtomicU64::new(0),
```

Comment:
Will move all of these metrics counter into a independent struct in https://github.com/risingwavelabs/risingwave/pull/7260